spec/domain/operations/notices/ivl_enr_notice_trigger_spec.rb

```diff
@@ -87,6 +87,18 @@
       end
     end
 
+    context '#build_addresses capitalize city/town' do
```

Comment:
Spec is not really testing the changes you've made. Your spec should fail when you reverted code changes.

Reply:
Hi @nisanthyaganti9 have made the changes on spec. Can you please review again? Thanks.

Reply:
@Rohithone 

```
let(:person) { FactoryBot.build(:person, addresses: [FactoryBot.build(:address, city: 'Washington dc')]) }
```

you can do this to avoid making changes in base factory